store/sqlstore/store_test.go

```diff
@@ -784,3 +784,41 @@ func TestMySQLReadTimeout(t *testing.T) {
 	_, err = store.GetMasterX().ExecNoTimeout(`SELECT SLEEP(3)`)
 	require.NoError(t, err)
 }
+
+func TestGetDBSchemaVersion(t *testing.T) {
+	testDrivers := []string{
+		model.DatabaseDriverPostgres,
+		model.DatabaseDriverMysql,
+	}
+
+	for _, driver := range testDrivers {
+		t.Run("Should return latest version number of applied migrations for "+driver, func(t *testing.T) {
+			t.Parallel()
+			settings := makeSqlSettings(driver)
+			store := New(*settings, nil)
+
+			version, err := store.GetDBSchemaVersion()
+			require.NoError(t, err)
+			require.NotZero(t, version)
```

Comment:
Wondering if we can read the migrations dir, find out the latest numbered sql file and match that with the value here? Or will that be too cumbersome? It seems like a `NotZero` test is too weak, but I don't want to have a super hacky check either. LMK.

Reply:
Yeah, it's doable. Same for below 👍